consumers/evm.md

```diff
@@ -4,13 +4,34 @@ description: Consume Pyth Network prices in applications on EVM-based chains
 
 # Pyth on EVM
 
+## How Pyth Works on EVM Chains
+
+Pyth prices are published on Solana and relayed to EVM chains using the [Wormhole Network](https://wormholenetwork.com/) as a cross-chain message passing bridge. The Wormhole Network observes when Pyth prices on Solana have changed and publishes an off-chain signed message attesting to this fact. This process is explained in more detail [here](https://docs.wormholenetwork.com/wormhole/).
+
+This signed price update message can then be submitted to the Pyth contract on the EVM networks, verifying the Wormhole message and updating the Pyth contract with the new price.
+
+Protocols integrating with Pyth can update the on-chain Pyth prices in two different ways. They can update the prices on demand or regularly,
```

Comment:
How about: 

Protocols integrating with Pyth are responsible for updating the on-chain Pyth price before using it. There are two different ways for protocols to perform these updates: